### ISSUE TITLE
Plasma cutter sounds

### ISSUE THREAD

**Issue by trerri** [goonstation/goonstation#168]
**Describe the bug**
Big pods have inconsistencies in playing the plasma cutter sound effect

**To Reproduce**
1. Put a plasma cutter in a 2x2 pod
2. Use the weapon in the 4 directions
3. The sound only plays when in the south or west directions

**Expected behavior**
The sound plays always because that's how sounds work

**Screenshots**
I cannot screenshot audio

**Comment by ZeWaka** [comment]
similar bug ages ago with rock drills

**Comment by Connie-Dun** [comment]
Theory: When a large pod is in space and fires a weapon, this block of code in ships.dm, lines 915-941:
```
if (shoot_dir == SOUTH || shoot_dir == WEST)
				var/obj/projectile/P = shoot_projectile_DIR(src, PROJ, shoot_dir)
				if (P)
					P.mob_shooter = user
					P.pixel_x = H * -5
					P.pixel_y = V * -5
			if (shoot_dir == SOUTH || shoot_dir == EAST)
				var/obj/projectile/P = shoot_projectile_DIR(get_step(get_turf(src), EAST), PROJ, shoot_dir)
				if (P)
					P.shooter = src
					P.mob_shooter = user
					P.pixel_x = H * 5
					P.pixel_y = V * -5
			if (shoot_dir == NORTH || shoot_dir == WEST)
				var/obj/projectile/P = shoot_projectile_DIR(get_step(get_turf(src), NORTH), PROJ, shoot_dir)
				if (P)
					P.shooter = src
					P.mob_shooter = user
					P.pixel_x = H * -5
					P.pixel_y = V * 5
			if (shoot_dir == NORTH || shoot_dir == EAST)
				var/obj/projectile/P = shoot_projectile_DIR(get_step(get_turf(src), NORTHEAST), PROJ, shoot_dir)
				if (P)
					P.shooter = src
					P.mob_shooter = user
					P.pixel_x = H * 5
					P.pixel_y = V * 5
```
sets both projectiles to say they originate from one of the other tiles of the pod. These tiles likely count as space, so when the projectile fires, the sound-silencing effect of space causes them to make no sound instead.
I tested it by firing plasma cutters and phasers from small and large pods, indoors and outdoors. There was only the issue of silence when either weapon was fired outdoors.

**Comment by Connie-Dun** [comment]
I don't know if I'm able to tie Issues to Pulls which fix them, but this accepted pull fixed this: https://github.com/goonstation/goonstation/pull/186

**Comment by ZeWaka** [comment]
> I don't know if I'm able to tie Issues to Pulls which fix them, but this accepted pull fixed this: #186

You can in the actual pull request. Type something like `Fixes #168` in the description.
